cmake/modules/SwiftSupport.cmake

```diff
@@ -71,31 +93,20 @@ function(_install_target module)
     set(swift swift)
   endif()
 
-  install(TARGETS ${module}
-    ARCHIVE DESTINATION lib/${swift}/${swift_os}
-    LIBRARY DESTINATION lib/${swift}/${swift_os}
-    RUNTIME DESTINATION bin)
+  install(TARGETS ${module})
```

Comment:
This wont install to the correct location on all platforms. The DLLs need to go to `bin` on Windows and the import libraries and static libraries into `lib`.

Reply:
Based on the [documentation](https://cmake.org/cmake/help/latest/command/install.html#targets) I believe this is already the default (scroll down for the table of default values).
Windows DLLs are `RUNTIME` artifacts, which get installed under `bin`.
Windows import and static libraries are `ARCHIVE` artifacts, which get installed under `lib`.

Reply:
I ran this branch on our internal build here: https://github.com/thebrowsercompany/swift-build/actions/runs/13001454471
It succeeded, including with packaging.